lib/rucio/tests/test_tpc.py

```diff
@@ -132,7 +134,13 @@ def test_tpc(containerized_rses, root_account, test_scope, did_factory, rse_clie
     assert fts_transfer_id is not None
     assert fts_transfer_status in ['SUBMITTED', 'ACTIVE']
```

Comment:
@rcarpa minor change, but this `assert` can be removed since you are checking this condition in the polling loop, and asserting this as a pre-condition would not really make a difference since `fts_transfer_status = None` before the loop begins ;)

Reply:
done